test/fixtures/poro.rb

```diff
@@ -266,3 +266,18 @@ def json_key
     raise StandardError, 'Intentional error for rescue_from test'
   end
 end
+
+CommentNestedSerializer = Class.new(ActiveModel::Serializer) do
+  attributes :body, :id
+end
+
+PostNestedSerializer = Class.new(ActiveModel::Serializer) do
+  attributes :title, :body, :id
+  has_many :comments, serializer: CommentNestedSerializer
+end
+
+AuthorNestedSerializer = Class.new(ActiveModel::Serializer) do
+  attributes :id, :name
+  has_many :posts, serializer: PostNestedSerializer
+end
```

Comment:
@joaomdmoura I think we should start following some convention for associating poros with tests. I think it's becoming append-only...